vos/vos/vos.py

```diff
@@ -2287,21 +2307,32 @@ def move(self, src_uri, destination_uri):
         destination_uri = self.fix_uri(destination_uri)
         with self.nodeCache.volatile(src_uri), self.nodeCache.volatile(
                 destination_uri):
-            return self.transfer(src_uri, destination_uri, view='move')
+            job_url = self.transfer(self.get_endpoints(src_uri).async_transfer,
+                                    src_uri, destination_uri, view='move')
+            # start the job
+            self.get_session(src_uri).post(
+                job_url + "/phase",
+                allow_redirects=False,
+                data="PHASE=RUN",
+                headers={'Content-type': "application/x-www-form-urlencoded"})
```

Comment:
Is it necessary to set this header?  I've never had to do this before.

Reply:
No effect so I've removed it.